lib/utils/display.js

```diff
@@ -17,7 +17,13 @@ const COLOR_PALETTE = ({ chalk: c }) => ({
   silly: c.inverse,
 })
 
-const LEVELS = ['timing', ...log.LEVELS].reduce((acc, key) => {
+const LOG_LEVELS = log.LEVELS.reduce((acc, key) => {
```

Comment:
Is this `KEYS` now?

Reply:
Yes, but not in this PR yet. I will add that as a follow-up to #7378